src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,49 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(
+    name,
+    health = 100,
+  ) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
 class Herbivore extends Animal {
-  // write your code here
+  constructor(
+    name,
+    health = 100,
+    hidden = false) {
+    super(name, health);
+
+    this.hidden = hidden;
+  }
+
+  hide() {
+    this.hidden = !this.hidden;
```

Comment:
```suggestion
    this.hidden = true;
```

Method hide should change hidden to true and not toggle it's value